projects/v2/src/app/store/sheet.state.ts

```diff
@@ -381,6 +395,9 @@ export class SheetState {
               rdfs_label: 'NONE',
             };
             row.anatomical_structures.unshift(newStructure);
+          }
+          asData = JSON.parse(JSON.stringify([...asData, ...res.data]));
```

Comment:
Is it really necessary to clone this data? Seems like just `[...asData, ...res.data]` would do the trick.